### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN023.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#69] [General]
# https://blog.3place.cn/3place/lmyn/LMYN023.html

https://blog.3place.cn/3place/lmyn/LMYN023.html

**Comment by picklecai** [comment]
第一天出门，需要师父叫醒做祭祀，哈哈哈哈！！！

**Comment by picklecai** [comment]
没出门时，对罗摩的父亲说：你这个儿子，天赋异禀，很适合（打罗刹）。
第一天刚出门，众友先教罗摩咒语。教完咒语，又给罗摩做心理建设。
第二天早上，叫醒罗摩，让他安排好一天的祭典。

这位老师，认可孩子本质上是优秀的，而后在具体事务行给予指导。不无视你的无知，也不扩大化你的无知。
这是真正爱你的好老师。

**Comment by picklecai** [comment]
爱神的净修林，叫鸯伽。
在鹿角仙人故事里，请鹿角仙人并把女儿散他嫁给仙人的，就是鸯伽王。
